### ISSUE TITLE
Add LKIF-Core prefix to http://prefix.cc

### ISSUE THREAD

**Issue by lewismc** [RinkeHoekstra/lkif-core#2]
Currently one of the main prefix look up's is http://prefix.cc/
LKIF-Core is not present and should be.

**Comment by RinkeHoekstra** [comment]
Good point. This is related to #1: if we change the namespace to setup permanent content negotiation, we'd need to register a different namespace than what's currently used. I think it is not a very good idea to continue to rely on the estrellaproject.org domain for hosting the ontology. What about a purl.org redirect?

**Comment by lewismc** [comment]
Hi Rinke. Do you have any suggestion for the purl?
I am in the process of sorting this out right now, if you can suggest then I'll get it set up, write the implementation into LKIF-Core send a pull request with the new purl configuration.

**Comment by lewismc** [comment]
Hi Rinke,
I'm dealing with this right now over at purl.org, I suggest to have the PURL as lkifcore.org
This keeps it nice and easy.
I'll update this thread once done.

**Comment by RinkeHoekstra** [comment]
Hi Lewis,

Great. Thanks for your effort! I'm currently overly busy with all kinds of
things entirely unrelated to LKIF core so I don't have any spare processing
cycles to really contribute.

-Rinke

**Comment by lewismc** [comment]
No bother Rinke. I realize this. I am using LKIF-Core and Metalex to compliment my Akmoa Ntoso Markup so I have an interest (and use case) for the time being.
It seems that the guys over at PURL are not too speedy about approving PURL domains so this may take some time.

**Comment by lewismc** [comment]
Hi @RinkeHoekstra, wow I just came across this old ticket. I am planning on possible using this ontology again. I'll kickstart the purl.org redirect again!

On another note... how quickly 3 entire years go!!!

**Comment by RinkeHoekstra** [comment]
Blast from the past indeed!

I’ve been having some issues with purl.org recently (cannot register new domains), but perhaps the w3id.org  service is an alternative. But that would require a namespace change hm....

-Rinke

---

Rinke Hoekstra
VU University Amsterdam/University of Amsterdam

**Comment by lewismc** [comment]
@RinkeHoekstra I think that the w3id service sounds like a very reasonable solution. It seems pretty easy to set up as well. I was also having issues setting up a purl recently. It seems like the service is unavailable which is a shame. What is your preference?

**Comment by lewismc** [comment]
Any preference here at all @RinkeHoekstra ?
